src/components/product-progress/dto/progress-format.enum.ts

```diff
@@ -1,46 +1,44 @@
-import { registerEnumType } from '@nestjs/graphql';
 import { stripIndent } from 'common-tags';
+import { EnumType, makeEnum } from '~/common';
 
-export enum ProgressFormat {
-  Numerator = 'Numerator',
-  Decimal = 'Decimal',
-  Percent = 'Percent',
-  Verses = 'Verses',
-  VerseEquivalents = 'VerseEquivalents',
-}
-
-registerEnumType(ProgressFormat, {
+export type ProgressFormat = EnumType<typeof ProgressFormat>;
+export const ProgressFormat = makeEnum({
   name: 'ProgressFormat',
   description: 'A format for a progress number',
-  valuesMap: {
-    Numerator: {
+  values: [
+    {
+      value: 'Numerator',
       description: stripIndent`
         The raw value that does not take into account the target value.
         This will be 0 <= # <= the product's \`progressTarget\` number.
         For example, # of Y complete
       `,
     },
-    Decimal: {
+    {
+      value: 'Decimal',
       description: stripIndent`
         A percent expressed as a decimal (0-1)
         For example, 0.# * 100 percent complete
       `,
     },
-    Percent: {
+    {
+      value: 'Percent',
       description: stripIndent`
-        A percent which already has already been multiplied by 100
-        For example, ##.#% complete
-      `,
+          A percent which already has already been multiplied by 100
+          For example, ##.#% complete
+        `,
```

Comment:
Can you revert these whitespace changes? (below too)
```suggestion
        A percent which already has already been multiplied by 100
        For example, ##.#% complete
      `,
```